provisioners/roles/postgres-ckan/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+# tasks file for postgres-ckan
+- template:
+    src=CentOS-Base.repo.j2
+    dest=/etc/yum.repos.d/CentOS-Base.repo
+  sudo: yes
+
+# - name: yum install postgres 9
+#   yum: name=http://yum.postgresql.org/9.3/redhat/rhel-6-x86_64/pgdg-centos93-9.3-1.noarch.rpm
+#   sudo: yes
+
+- name: ensure PGDG Yum repo is installed
+  shell: rpm -Uvh http://yum.postgresql.org/9.3/redhat/rhel-6-x86_64/pgdg-centos93-9.3-1.noarch.rpm 
+    creates=/etc/yum.repos.d/pgdg-93-centos.repo
+  sudo: yes
+
+- name: install postgres dependencies
+  yum: name={{item}}
+  with_items: 
+    - postgresql93-devel
+    - postgresql93-server
+    - postgresql93-libs
+    - python-psycopg2
+  sudo: yes
+
+- name: determine if db initialized
+  stat: path=/var/lib/pgsql/9.3/data/pg_hba.conf
```

Comment:
I think I'll just split it out into two separate roles as per Dev's suggestion below.